### DISCUSSION TITLE
ConnectedComponent Area-Threshold single value vs range

### DISCUSSION THREAD

**Discussion by Oninaig** [ImageMagick/ImageMagick#2678] [Help] [answered]
According to the documentation:

```connected-components:area-threshold=value``` ```Merges any region with area smaller than value into its surrounding region or largest neighbor.```

This makes sense for single values. For example, if you said 450 it would mean that any region with area smaller than 450 would be merged into its surrounding region or largest neighbor.

I am a little confused, however, when you use a range such as 450-1000. Does this apply the same logic to _every_ value between 450-1000? How would that be different than just saying 1000?

**Comment by fmw42** [comment, marked as answer]
It has recently been changed to allow a range. The docs apparently need changing. It means it accepts only values between the range. Any value  below or above the range will be excluded.

**Reply by Oninaig** [reply]
Right, its a maximum. Got it :thumb:
